src/authentication/create-account-method/index.tsx

```diff
@@ -23,14 +23,15 @@ export const CreateAccountMethod: React.FC<CreateAccountMethodProps> = ({ stage,
     { key: 'email', label: 'Email' },
   ];
 
-  const selectedOption = stage === RegistrationStage.WalletAccountCreation ? 'web3' : 'email';
   const isWalletAccountCreationStage = stage === RegistrationStage.WalletAccountCreation;
+  const selectedOption = isWalletAccountCreationStage ? 'web3' : 'email';
+  const shouldRenderToggleGroup = isConnecting && isWalletAccountCreationStage;
 
   return (
     <div {...cn('', isConnecting && isWalletAccountCreationStage && 'is-connecting')}>
       <h3 {...cn('heading')}>Create Account</h3>
 
-      {!isConnecting && (
+      {!shouldRenderToggleGroup && (
```

Comment:
This is confusing to me. "If not should render...then render"?

Reply:
Oops sorry yeah that does not make any sense. Updated.